examples/complex/esp32/Gateway/README.md

```diff
@@ -0,0 +1,45 @@
+# ESP32 Gateway demo
+
+This demo uses two inexpensive and readily available ESP32's to send
+telemetry data, one ESP32 acting as a gateway to Google Cloud IoT core and another ESP32
+acting as the delegate device that sends data to the gateway.The gateway will send the
+data to Google Cloud IoT core and publish to PubSub. For example, you can send temperature
+data from multiple delegate devices to Google Cloud and store that information to see
+any drastic temperature changes in a room.
+
+**Disclaimer** This demo is still in a rough state, it can crash the ESP32 and is
+offered with limited support.
+
+Some of the SerialBT code is based on [Espressif](https://github.com/espressif/arduino-esp32/tree/master/libraries/BluetoothSerial/examples/) examples
+that shows you how to connect two devices over serial bluetooth.
+
+## Configuration
+You need to configure the devices to connect over bluetooth in `ESP32_delegate.ino`.
+Make sure that the device_id you enter matches the device_id on the Google Cloud IoT core.
+The example configurations reflect models encountered during testing but you
+may need to add your own based on the data sheet for your device.
+
+Update the values in [ciotc_config.h](ciotc_config.h) so that they match the
+configuration for your gateway device and delegate devices.
+
+## Running the demo
```

Comment:
Add instruction for adding devices in the console, maybe collapse the configuration and running sections into a numbered list:

1. Add devices in the console.
2. Update `ciotc_config.h` in Esp32-Gateway to match the devices.
3. Flash the gateway device with the project sources in Esp32-Gateway.
4. Flash the delegate device(s) with the project sources in Esp32-delegate.
5. Restart both devices...